.github/workflows/android.yml

```diff
@@ -30,6 +30,8 @@ on:
 jobs:
   build:
     runs-on: ubuntu-22.04
+    env:
+      JAVA_HOME: /usr/lib/jvm/java-17-openjdk-amd64/
```

Comment:
Is there a less dumb way of doing this?

Reply:
For Principia's Android CI we begun using `actions/setup-java` to setup the JDK:

```yaml
      - name: Set up JDK 17
        uses: actions/setup-java@v4
        with:
            distribution: 'temurin'
            java-version: '17'
```

Temurin is the JDK distribution that is included by default in the Github Workflows runner afaik (defaulting to version 11)